src/java.base/macosx/native/libjli/java_md_macosx.m

```diff
@@ -273,15 +275,23 @@
     sPreferredJVMType = strdup(prefJVM);
 }
 
+#ifdef __IOS__
+static jboolean awtLoaded = 0;
```

Comment:
How come this change is needed? Why do `BOOL` not work on iOS? Or contrary, could `jboolean` work on macOS as well?

Reply:
I agree, it seems unused. I can only find a read in `JLI_NotifyAWTLoaded` but that is never called. 

I guess we should file a bug on mainline to remove this code. Question is if we should keep this hack in mobile for the time being. I guess we can do that, if it is needed to make it compile. A fix in mainline will likely trigger a manual merge later on.

Reply:
Right, the skara bot will probably fail to auto-merge the upstream fix (once it's integrated), but it will auto-create a PR that requires a small manual commit, and then the automated process continues -- this happened in the past, and it works very nice.